src/main/java/MyTasks.java

```diff
@@ -57,7 +56,6 @@ public static void findLongestWordInFile() throws FileNotFoundException {
             Scanner scanner = new Scanner(new File("C:\\Users\\mmazu2\\studying\\text.txt"));
```

Comment:
There is should be a relative path to the text file. E.g. start with 'System.getProperty("user.dir")' and then put a file destination in the framework